tests/test_ocs_basic_install.py

```diff
@@ -59,7 +60,13 @@ def run(**kwargs):
     else:
         log.info("Downloading openshift installer")
         ver = config.get('installer-version')
-        url = f"https://github.com/openshift/installer/releases/download/{ver}/openshift-install-linux-amd64"
+        if platform.system() == "Darwin":
+            os_type = "darwin"
+        elif platform.system() == "Linux":
+            os_type = "linux"
+        else:
```

Comment:
Should we raise an Exception here instead of setting to None? The download will fail anyways.

Reply:
I thought of that, then there are 3 options Darwin, Linux and Windows. Windows will exit sooner, I can still add an exception here we can check when it reaches here.

Reply:
As long as we are just raising an exception, the runner should handle the rest for us. It will except it, log, fail the test, and move on if not configured to abort on failure.